internal/guest/README.md

```diff
@@ -0,0 +1,4 @@
+# Background
+The packages in this directory comprise most of the code used to build the guest side agent for Linux Hyper-V containers on Windows (LCOW). The two binaries of importance that are built from this code are the [main guest agent](../../cmd/gcs/main.go) that facilitates communication between the host and guest and a [generic tools binary](../../cmd/gcstools/main.go) for additional functionality for device scenarios. The guest agent is designed to run inside a custom Linux OS for supporting Linux container payloads.
+
+As most of the rest of the repository is designed in part to run on Windows, the guest package tries to serve as a separation between the two. Any feature work or Linux specific functionality should end up here.
```

Comment:
Not sure I get this line.

Reply:
The Linux specific functionality bit?

Reply:
I think maybe just saying "any feature work specific to LCOW should live here" or something.

Reply:
donezo